jerry-core/vm/vm.c

```diff
@@ -1102,11 +1107,18 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
         case VM_OC_SET_GETTER:
         case VM_OC_SET_SETTER:
         {
+          JERRY_ASSERT (byte_code_start_p[0] == CBC_EXT_OPCODE);
+#ifndef CONFIG_DISABLE_ES2015_CLASS
+          opfunc_set_accessor (VM_OC_GROUP_GET_INDEX (opcode_data) == VM_OC_SET_GETTER,
+                               stack_top_p[(byte_code_start_p[1] > CBC_EXT_SET_SETTER) ? -2 : -1],
```

Comment:
I would use a `const int index` just like above instead of duplicating the code.